docs/redirects.asciidoc

```diff
@@ -308,3 +308,7 @@ This content has moved. Refer to <<discover-view-surrounding-documents>>.
 
 This content has moved. Refer to <<string-field-formatters>>.
 
+[role="exclude",id="alerting-setup-prerequisites"]
```

Comment:
@gchaps I added a section to redirects but am still failing CI. 

```
INFO:build_docs:Bad cross-document links:
11:42:23 INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/observability/master/create-alerts.html contains broken links to:
11:42:23 INFO:build_docs:   - en/kibana/master/alerting-getting-started.html#alerting-setup-prerequisites
```

Any ideas?

Reply:
Unfortunately, redirects don't work for section-level links from other books.  We'll need to open a PR on the observability-docs repo to fix that link.

Reply:
@lcawl Thanks! What is the order of operations here? Should I submit the PR and get it merged first then open a PR in that repo?

Reply:
I think the simplest method (to avoid a chicken-and-egg scenario) might be:
(1) leave the anchor in its existing location (but with the text removed) and refrain from creating the redirect page for now. This should hopefully pass the build checks.
(2) open a PR in observability-docs to update the link to use the new page.
(3) open a PR in kibana to remove the lingering anchor (at this point you shouldn't need a redirects page entry, but I guess it doesn't hurt).

If you want my help with any of those changes or I haven't explained this sufficiently, just let me know!

Reply:
@lcawl That makes sense, thank you!